tools/ci/scripts/buildAllLocal.ps1

```diff
@@ -0,0 +1,23 @@
+$MSBuild = "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\amd64\MSBuild.exe"
+$UnityPath = "C:\Program Files\Unity\Hub\Editor\2018.3.14f1\Editor\Unity.exe"
```

Comment:
It's not great that those paths are hard coded. If we can't detect those, perhaps it might be a good idea to set up a shared config file somewhere? This is yet another place where Unity version needs to be updated if build requirements change.
TBH, if you don't want to do spend time on doing this now perhaps it's worth adding a TODO comment or creating a work item for this.

Reply:
when i set up some of the buildNativeLocal.ps1 logic I had these as fallback values if no value was specified. Long term it seems like we could use some of the searching functionality that exists for unity in the pipeline build to find these executables. I'll take another pass on this before checking in.